Guides/Development.md

```diff
@@ -70,7 +70,7 @@ Documentation comments should generally be complete sentences and should end wit
 
 Our documentation site is automatically generated from the source code using [Jazzy](https://github.com/realm/jazzy#installation). We regenerate it via our release script each time we release a new version of the driver.
 
-If you'd like to preview how new documentation you've written will look when published, you can regenerate it by running `make documentation` and then inspecting the generated HTML files in `/docs`.
+If you'd like to preview how new documentation you've written will look when published, you can regenerate it by running `.etc/generate-docs.sh` and then inspecting the generated HTML files in `/docs`.
```

Comment:
typo: should be `./etc/generate-docs.sh`

Reply:
good catch, fixed

Reply:
I think you forgot to push the commit